src/cljs/chronograph_web/pages/create_organization/events.cljs

```diff
@@ -0,0 +1,71 @@
+(ns chronograph-web.pages.create-organization.events
+  (:require [re-frame.core :as rf]
+            [chronograph-web.db :as db]
+            [chronograph-web.http :as http]
+            [chronograph-web.events.routing :as routing-events]
+            [ajax.core :as ajax]
+            [day8.re-frame.http-fx]
+            [clojure.spec.alpha :as s]))
+
+(def ^:private create-organization-uri "/api/organizations")
+
+(def ^:private root-path :create-organization)
+(def ^:private status-path [root-path :status])
+(defn- form-params-path [k] [root-path :form-params k])
+
+(defn- form-errors-path
+  ([] [root-path :form-errors])
+  ([k] (conj (form-errors-path) k)))
+
+(def ^:private field-spec
+  {:slug :organizations/slug
+   :name :organizations/name})
+
+(defn- update-form-error [db k v]
```

Comment:
The errors are a pure function of the form input state, we should avoid storing them separately in the DB. This logic can go into the form subscription.

Reply:
Makes sense, but not sure how to rework it. Will take a shot.

Reply:
We've reworked this into the view. The errors are computed from state directly and not stored there.